fastify.js

```diff
@@ -241,6 +227,18 @@ function fastify (options) {
     }
   })
 
+  ;[
+    { f: 'prefix', s: kRoutePrefix },
+    { f: 'validatorCompiler', s: kValidatorCompiler },
+    { f: 'serializerCompiler', s: kSerializerCompiler }
+  ].forEach(({ f, s }) => {
+    Object.defineProperty(fastify, f, {
+      get: function () {
+        return this[s]
+      }
+    })
+  })
```

Comment:
I do like the functionality of this code, but I think I'd prefer these to be hard coded if possible. It is only 3 things and the complexity of being in a forEach loop (which are fairly slow) feels unnecessary to me.